UPGRADE.md

```diff
@@ -2,6 +2,14 @@
 
 ## dev-develop
 
+### Address country is nullable
+
+To make it easier to migrate data the country in the address entity is now nullable in sulu.
+
+```bash
+app/console doctrine:schema:update --force
```

Comment:
I thought we don't add this anymore? Maybe you can just add a single section for the database change, otherwise we'll end up with multiple `doctrine:schema:update`s in the UPGRADE.md